src/entities/amm/amm.ts

```diff
@@ -1494,15 +1497,20 @@ export class AMM {
         return borrowApy;
       }
 
-      case 7: {
+      case 7:
+      case 8: {
```

Comment:
While I get that we have code in common, I think this should be avoided by refactoring the entire switch statement. Therefore, I would say that for now, we should have separate cases for distinct protocols.

Reply:
gotcha [1655d3a](https://github.com/Voltz-Protocol/v1-sdk/pull/213/commits/1655d3adeef748d7055d1afc6817eb0907f34328)